src/main/webapp/i18n/de/lectureUnit.json

```
@@ -114,6 +114,33 @@
             "created": "Eine neue Dateieinheit wurde erstellt",
             "updated": "Dateieinheit aktualisiert",
             "notReleasedTooltip": "Nur sichtbar für Tutor:innen und Lehrende. Veröffentlichungsdatum:",
+            "createAttachmentUnits": {
+                "noUnitDetected": "Es wurde keine Einheit erkannt und konnte nicht automatisch geteilt werden. Bitte fügen Sie Informationen für jede Einheit hinzu!",
+                "pageTitle": "Prozesseinheiten",
```

```suggestion
                "pageTitle": "Einheiten",
```
Someone more familiar with the lecture units might be able to come up with a better name/one that is consistent with the remaining ones for this area of Artemis.